source/help/getting-started/setting-your-status-availability.rst

```diff
@@ -43,6 +43,19 @@ When you set your availability, an availability icon displays next to your name.
 
 Other members can see your availability anywhere they can see your name, such as the channel sidebar, within conversations, and within Direct Messages. 
 
+Pausing All Notifications
+~~~~~~~~~~~~~~~~~~~~~~~
+
+From Mattermost v5.36, you can pause all notifications for a period of time when you need to concentrate or when you're unavailable.  
```

Comment:
DND was available before v5.36. In v5.36 the timeout option was added. Also propose calling this section Do Not Disturb - since the page is about statuses, rather than notifications